kalite/distributed/templates/distributed/base.html

```diff
@@ -182,8 +182,8 @@
                               <form class="navbar-form" action="{% url 'search' %}" method="get" id="search-box" role="search">
                                 {% comment %}translators: this will appear in the navigation bar. please try to keep it as short as possible.{% endcomment %}
                                 <div class="input-group">
-                                  <input type="text" name="query" id="search" class="form-control" placeholder="{% trans 'Topic, video, exercise...' %}" />
-                                  <button id="search-button" class="btn btn-default" type="submit"><i class="glyphicon glyphicon-search"></i></button>
+                                  <input type="text" name="query" id="search" class="form-control" title="Type search term here" placeholder="{% trans 'Topic, video, exercise...' %}" />
+                                  <button id="search-button" class="btn btn-default" type="submit" value="Search"><i class="glyphicon glyphicon-search"></i></button>
```

Comment:
I don't know enough about how screen readers work to understand the purpose of the 'value' attribute here - can you explain?